libraries/cyclestreets-view/src/main/java/net/cyclestreets/routing/Journey.kt

```diff
@@ -200,6 +208,29 @@ class Journey private constructor(wp: Waypoints? = null) {
                 journey.elevation.add(sdo.segmentProfile)
             }
         }
+        // For circular routes.
+        private fun populatePois(jdo: JourneyDomainObject, context: Context) {
+            // For circular route pois, the id isn't returned in the API, so will create an artificial one.
+            var id = 0
+            for (poi in jdo.pois) {
+                val circularRoutePoi = POI(id,
+                        poi.name,
+                        "",
```

Comment:
Don't we have the other fields available for the constructor?

Reply:
See note 12 in "Other notes" above.  Only limited info is returned for Circular Route POIs.

Reply:
The documentation just lists these 6 attributes: longitude, latitude, sequenceId, poitypeId, name, website 
https://www.cyclestreets.net/api/v1/leisure/